salt/_modules/metalk8s_service_configuration.py

```diff
@@ -2,9 +2,11 @@
 
 import logging
 import yaml
+import json
 
 import salt
 from salt.exceptions import CommandExecutionError
+from typing import Literal
```

Comment:
That's only available to Python 3.8, right? And buildchain is running with 3.6 in CI, no?